easybuild/easyblocks/generic/rpackage.py

```diff
@@ -187,7 +187,14 @@ def run(self):
             lib_install_prefix = os.path.join(rhome, 'library')
         else:
             # extension is being installed in a separate installation prefix
-            lib_install_prefix = self.installdir
+            lib_install_prefix = os.path.join(self.installdir, 'library')
+            try:
+                os.mkdir(lib_install_prefix)
+            except OSError, err:
+                if os.path.isdir(lib_install_prefix):
+                    pass
+                else:
+                    raise EasyBuildError("Failed to create library directory for R packages: %s", err)
```

Comment:
the whole `try`/`except` block can be replaced by calling the `mkdir` function from `easybuild.tools.filetools`?

Reply:
@pneerincx ping on this remark?